module/advancement/advancement-config.mjs

```diff
@@ -101,4 +114,86 @@ export default class AdvancementConfig extends FormApplication {
     }, {});
   }
 
+  /* -------------------------------------------- */
+  /*  Drag & Drop for Item Pools                  */
+  /* -------------------------------------------- */
+
+  /**
+   * Handle deleting an existing Item entry from the Advancement.
+   * @param {Event} event        The originating click event.
+   * @returns {Promise<Item5e>}  The updated parent Item after the application re-renders.
+   * @protected
+   */
+  async _onItemDelete(event) {
+    event.preventDefault();
+    const uuidToDelete = event.currentTarget.closest("[data-item-uuid]")?.dataset.itemUuid;
+    if ( !uuidToDelete ) return;
+    const items = foundry.utils.getProperty(this.advancement.data.configuration, this.options.dropKeyPath);
+    const updates = { configuration: await this.prepareConfigurationUpdate({
+      [this.options.dropKeyPath]: items.filter(uuid => uuid !== uuidToDelete)
+    }) };
+    console.log(updates);
+    await this.advancement.update(updates);
+    this.render();
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  _canDragDrop() {
+    return this.isEditable;
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  async _onDrop(event) {
+    if ( !this.options.dropKeyPath ) return console.error(
+      "AdvancementConfig#options.dropKeyPath must be configured or #_onDrop must be overridden to support"
+      + " drag and drop on advancement config items."
+    );
+
+    // Try to extract the data
+    let data;
+    try {
+      data = JSON.parse(event.dataTransfer.getData("text/plain"));
+    } catch(err) {
+      return false;
+    }
+
+    if ( data.type !== "Item" ) return false;
+    const item = await Item.implementation.fromDropData(data);
+
+    const verified = this._verifyDroppedItem(event, item);
```

Comment:
I think this method name might be more clear with two changes:
1. `validate` instead of `verify`
2. The `_validateDroppedItem` method would `throw` and Error if invalid rather than returning `false`